syntax = "proto3";

package PbConfigs;

message Weather
{
    double temperature = 1;
    double humidity = 2;
}

message SessionConfig
{
    string circuitId = 1;
    string raceId = 2;
    string test = 3;
    string driverId = 4;
    string date = 5;
    string time = 6;
    Weather weather = 7;
    string notes = 8;
    double canlibVersion = 9;
}